easybuild/framework/easyblock.py

```diff
@@ -1642,8 +1642,8 @@ def make_module_step(self, fake=False):
 
         txt = ''
         txt += self.make_module_description()
-        txt += self.make_module_extend_modpath()
         txt += self.make_module_dep()
+        txt += self.make_module_extend_modpath()
```

Comment:
from the top of my head, I don't think it matters... if used to matter, but that's because we weren't handling things correctly, that was fixed post v1.14

let's see if the unit tests prove me wrong ;)